Library/Formula/natalie.rb

```diff
@@ -0,0 +1,23 @@
+class Natalie < Formula
+  homepage "https://github.com/krzyzanowskim/Natalie"
+  url "https://github.com/krzyzanowskim/Natalie/archive/0.1.tar.gz"
+  sha256 "943dab1b7ac16555ec3047983a3c397f7ebe7559d3c098c02a6ee6d8e966655e"
+  head "https://github.com/krzyzanowskim/Natalie.git"
+
+  depends_on :xcode => "6.0"
+
+  def install
+    bin.install "natalie.swift"
+    etc.install Dir["NatalieExample"] unless File.exists? (etc/"NatalieExample")
+  end
+
+  test do
+    example_path = "#{etc}/NatalieExample"
+    output_path = "#{testpath}/Storyboards.swift"
```

Comment:
Make output_path a Pathname with `output_path = testpath/"Storyboards.swift"`

and then you can do `output_path.write(generated_code)` below.